src/components/SignIn/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import HeaderLayout from "../../layouts/HeaderLayout";
+import SignInForm from "./SignInForm";
+
+import { Flex, Image, Text } from "@chakra-ui/react";
+
+const SignIn = () => {
+  return (
+    <HeaderLayout title="Sign In">
```

Comment:
Head tag should be on the page file cos it is not related to any component.

Reply:
done